bbb_presentation_video/events/tldraw.py

```diff
@@ -1,31 +1,59 @@
-# SPDX-FileCopyrightText: 2022 BigBlueButton Inc. and by respective authors
+# SPDX-FileCopyrightText: 2024 BigBlueButton Inc. and by respective authors
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 import json
-from typing import Dict, List, Optional, TypedDict
+from typing import Any, Collection, Dict, List, Optional, Sequence, TypedDict, Union
 
 from lxml import etree
 
 from bbb_presentation_video.events.helpers import Position, xml_subelement
 
 
 class StyleData(TypedDict, total=False):
-    color: str
     dash: str
     font: str
     isFilled: bool
     scale: float
-    size: str
     textAlign: str
+    opacity: float
+    isComplete: bool
+    fill: str
+    color: str
+    size: str
+    isClosed: bool
+    segments: List[Dict[str, Sequence[Collection[str]]]]
 
 
 class HandleData(TypedDict, total=False):
+    bindingId: str
+    canBind: bool
+    canSnap: bool
     id: str
-    index: float
+    index: Union[str, int]
```

Comment:
In the newer Tldraw version, shapes (e.g., rectangles) have indexes such as  `"index": "a1"`, which is why I set it to `str`. In testing with BBB 2.7, I only encountered integers, but since I cannot assure this is the case for previous versions I've now set it to `Union[str, float]`.